munimap/views/api.py

```diff
@@ -1,14 +1,15 @@
-from __future__ import absolute_import
+
 
 import logging
 
 
 from flask import (
     Blueprint,
     current_app,
-    request,
+    Request,
```

Comment:
Is this import intentional?

Reply:
No, `Request` is not being used. Removed